src/cljs/main/broadfcui/page/library/library_page.cljs

```diff
@@ -196,6 +196,7 @@
                                [:p {} "After dbGaP approves your application please link your eRA
                                        Commons ID in your FireCloud profile page."])])])))}
          {:href (str (when (common/has-terra-return?) (config/terra-url))
+                     (when (common/has-firecloud-return?) (config/firecloud-terra-url))
                      (nav/get-link :workspace-summary (common/row->workspace-id data)))})))
```

Comment:
this should use `cond` instead of the stacked `when`s, something like:
```
         {:href (str (cond
                       (common/has-terra-return?) (config/terra-url)
                       (common/has-firecloud-return?) (config/firecloud-terra-url))
                     (nav/get-link :workspace-summary (common/row->workspace-id data)))})))
```

Reply:
also, not a result of your  changes - the URLs generated by this syntax work, but are incorrect in that they're missing a / before the hash, e.g.:
https://firecloud.terra.bio#workspaces/namespace/name
should really be
https://firecloud.terra.bio/#workspaces/namespace/name

Reply:
<img width="579" alt="Screen Shot 2019-04-16 at 2 40 31 PM" src="https://user-images.githubusercontent.com/42386483/56235618-c380de00-6055-11e9-9d34-0e0dcf181c3d.png">

<img width="731" alt="Screen Shot 2019-04-16 at 2 42 16 PM" src="https://user-images.githubusercontent.com/42386483/56235737-0642b600-6056-11e9-8b24-663a1fe62ea8.png">


I'm seeing it with the / in both places. Is that by accident though?

Reply:
the url generated by the code is missing the / ... you can see this is you inspect the DOM via dev tools. Once a user clicks on it, the browser (Chrome at least) normalizes the url and inserts the missing /. So I don't think it's a big deal  and I don't think you should hold the PR for this fact ... but in the big picture we shouldn't rely on browsers to fix up the url for us.

Reply:
Ah I see, okay that's why I wasn't noticing it. Cool! Learned a thing--I'll fix it.